src/BitStringAddresses/BitStringAddresses.jl

```diff
@@ -18,8 +18,8 @@ export OccupationNumberFS
 export BoseFSIndex, FermiFSIndex
 export BitString, SortedParticleList
 export num_particles, num_modes, num_components
-export find_occupied_mode, find_mode, occupied_modes, is_occupied, num_occupied_modes
-export excitation, near_uniform, OccupiedModeMap, OccupiedPairsMap
+export find_occupied_mode, find_mode, occupied_modes, unoccupied_modes, is_occupied, num_occupied_modes, num_unoccupied_modes
+export excitation, near_uniform, ModeMap, OccupiedPairsMap, occupied_mode_map, unoccupied_mode_map
```

Comment:
```suggestion
export excitation, near_uniform, OccupiedPairsMap, occupied_mode_map, unoccupied_mode_map
```
I lean towards not exporting `ModeMap`, as the user typically would not have to interface with it.

Reply:
This will require explicitly importing the symbol in `Hamiltonians/Hamiltonians.jl`:
```julia
using ..BitStringAddresses: ModeMap
```